install_config/registry/securing_and_exposing_registry.adoc

```diff
@@ -293,3 +293,92 @@ not the route name and port. See `oc get imagestreams` for details.
 ====
 In the `<host>/test/busybox` example above, `test` refers to the project name.
 ====
+
+
+[[access-secure-registry-by-exposing-route]]
+=== Exposing a Secure Registry
+
+Instead of logging in to the registry from within the {product-title} cluster,
+you can gain external access to it by first
+xref:securing-the-registry[securing the registry] and then
+xref:exposing-the-registry[exposing the registry].
+This allows you to log in to the registry from outside the cluster using the
+route address, and to tag and push images using the route host. 
+
+After securing and exposing the registry, update your *_/etc/sysconfig/docker_* file by adding the port number to the `OPTIONS` entry. For example:
+
+----
+OPTIONS='--selinux-enabled --insecure-registry=172.30.0.0/16 --insecure-registry registry.ose-node.openshift.com:80'
```

Comment:
The secured route will use the 443 port by default.
BTW, seems if we secured the docker registry, the option `--insecure-registry registry.ose-node.openshift.com:80` is not needed in the docker daemon config `/etc/sysconfig/docker`.